test/cljs/rems/administration/test_create_catalogue_item.cljs

```diff
@@ -46,3 +46,82 @@
+                                :enabled true
+                                :id 8
+                                :licensetype "text"
+                                :localizations {:en {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Second License"}}}]
+                    :modifieruserid "owner"
+                    :organization {:organization/id "nbn"
+                                   :organization/name {:en "NBN"
+                                                       :fi "NBN"
+                                                       :sv "NBN"}
+                                   :organization/short-name
+                                   {:en "NBN en"
+                                    :fi "NBN fi"
+                                    :sv "NBN sv"}}
+                    :owneruserid "owner"}
+                   {:resid "y"
+                    :licenses [{:archived false
+                                :enabled true
+                                :id 8
+                                :licensetype "text"
+                                :localizations {:en {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Performance License"}}}
+                               {:archived false
+                                :enabled true
+                                :id 8
+                                :licensetype "text"
+                                :localizations {:en {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Second License"}}}]
+                    :organization {:organization/id "nbn"
+                                   :organization/name {:en "NBN"
+                                                       :fi "NBN"
+                                                       :sv "NBN"}
+                                   :organization/short-name {:en "NBN en"
+                                                             :fi "NBN fi"
+                                                             :sv "NBN sv"}}}
+                   {:resid "y"
+                    :licenses [{:archived false
+                                :enabled true
+                                :id 8
+                                :licensetype "text"
+                                :localizations {:en {:attachment-id nil
+                                                     :textcontent "Be fast."
+                                                     :title "Performance License"}}}]}
+                   {:resid "u"}
+                   {:resid "z"}]
+        counts (frequencies (map :resid resourses))]
+    (testing "sorting resources"
+      (is (not (empty? (map resourse-dropdown resourses))))
```

Comment:
this test case looks a bit weird: it's calling `resourse-dropdown` with only one parameter, when it really wants 3, right?